<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Avogadro::QtPlugins::TemplateToolWidget</class>
 <widget class="QWidget" name="Avogadro::QtPlugins::TemplateToolWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="elementLabel">
     <property name="text">
      <string>Element:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="elementComboBox">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <item>
      <property name="text">
       <string>Hydrogen</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Carbon</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Iron</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cobalt</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="coordinationLabel">
     <property name="text">
      <string>Coordination:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="coordinationComboBox">
     <property name="currentText">
      <string>6: Octahedral</string>
     </property>
     <property name="currentIndex">
      <number>7</number>
     </property>
     <property name="maxCount">
      <number>2147483646</number>
     </property>
     <item>
      <property name="text">
       <string>1: Linear</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2: Linear</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3: Trigonal Planar</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4: Tetrahedral</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4: Square Planar</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5: Trigonal Bipyramidal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5: Square Pyramidal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6: Octahedral</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6: Trigonal Prism</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7: Pentagonal Bipyramidal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8: Square Antiprism</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="ligandLabel">
     <property name="text">
      <string>Ligand:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="ligandComboBox">
     <item>
      <property name="text">
       <string>Ammine</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Aqua</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Carbonyl</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Phosphine</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Thiol</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Methyl</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ethyl</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Propyl</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Isopropyl</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>t-Butyl</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="preview">
     <property name="text">
      <string>octahedral</string>
     </property>
     <property name="icon">
      <iconset theme=":/icons/coord/6-octahedral.png">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="template.qrc"/>
 </resources>
 <connections/>
</ui>
